src/App/App.js

```diff
@@ -170,42 +164,38 @@ class App extends Component {
     this.setState( {cardAreaView: view})
   }
 
-  removeFaveListItem = (indexToRemove) => {
-    const [...newFaveDecks] = this.state.userFaveDecks;
-    newFaveDecks.splice(indexToRemove, 1)
-    this.setState({ userFaveDecks: newFaveDecks },
-      localStorage.setItem('userFaveDecks', JSON.stringify(newFaveDecks)));
+  addToWishlist = (card) => {
+    card.wishListCount = 1;
+    const newWishList = this.state.wishList.concat([card]);
+    this.saveArray('wishList', newWishList);
   }
 
-  removeWishListItem = (indexToRemove) => {
-    const [...newWishList] = this.state.wishList;
-    newWishList.splice(indexToRemove, 1)
-    this.setState({ wishList: newWishList },
-      localStorage.setItem('wishList', JSON.stringify(newWishList)));
+  addToFaveDecks = (deck) => {
+    const newFaveDecks = this.state.faveDecks.concat([deck]);
+    this.saveArray('faveDecks', newFaveDecks);
   }
 
   render() {
     return (
       <div className="app">
         <Aside
           addUserCard={this.addUserCard}
-          removeUserCard={this.removeUserCard}
-          setCardCount={this.setCardCount}
           userCardsData={this.state.userCardsData}
           cards={this.state.cards}
           compareBuilds={this.compareBuilds}
           asideView={this.state.asideView}
-          userFaveDecks={this.state.userFaveDecks}
-          removeFaveListItem={this.removeFaveListItem}
+          faveDecks={this.state.faveDecks}
           wishList={this.state.wishList}
-          removeWishListItem={this.removeWishListItem} />
+          saveArray={this.saveArray}/>
```

Comment:
The `saveArray()` method removes the need for `removeUserCard()`, `setCardCount()`, `removeFaveListItem()`, and `removeWishListItem()`.